docker/Dockerfile

```diff
@@ -0,0 +1,41 @@
+ARG DISTRO=humble
+FROM ros:${DISTRO}
+ARG DISTRO
+
+SHELL ["/bin/bash", "-c"]
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+USER root
+
+# Install
+RUN apt update \
+  && apt upgrade -y \
+  && apt install -y cmake curl git python3
+
+# Install and configure ROS infrastructure tools
+RUN apt update \
+  && apt install -y python3-vcstool python3-colcon-common-extensions python3-rosdep \
+  && rosdep update
+
+# Install the dependency repositories
+# Use a tmpfs mount for the workspace so as not to unnecessarily copy files into the final image
+# Bind mount the source directory so as not to unnecessarily copy source code into the docker image
+ARG WORKSPACE_DIR=/tmpfs/noether_ros
+ARG INSTALL_DIR=/opt/noether_ros
+RUN mkdir -p ${INSTALL_DIR}
+
+RUN --mount=type=tmpfs,target=${WORKSPACE_DIR} --mount=type=bind,target=${WORKSPACE_DIR}/src/noether_ros \
+  source /opt/ros/${DISTRO}/setup.bash \
+  && cd ${WORKSPACE_DIR} \
+  && vcs import src < src/noether_ros/dependencies.repos --shallow \
+  && rosdep install \
+    --from-paths ${WORKSPACE_DIR}/src \
+    -iry \
+    --skip-keys libvtk \
```

Comment:
If I remember correctly, this skip is necessary because VTK imports its own sql version, which can cause headaches when used alongside QT. Could you leave a comment documenting that reason, so that this can get cleaned up if the issue is resolved in the future? (And so that a 'helpful' engineer doesn't 'clean it up' in the future BEFORE it is fixed!)

Reply:
> If I remember correctly, this skip is necessary because VTK imports its own sql version

No; this is needed because the key `libvtk` still installs VTK7 on 22.04 and 24.04 and we're using VTK9 now